src/main/plugin/iso19139.ca.HNAP/templates/hnap_nonspatial-english.xml

```diff
@@ -320,13 +338,13 @@
                   <gco:CharacterString/>
                   <gmd:PT_FreeText>
                      <gmd:textGroup>
-                        <gmd:LocalisedCharacterString locale="#fra"/>
+                        <gmd:LocalisedCharacterString locale=""/>
                      </gmd:textGroup>
                   </gmd:PT_FreeText>
                </gmd:keyword>
                <gmd:type>
                   <gmd:MD_KeywordTypeCode codeList="https://schemas.metadata.geo.ca/register/napMetadataRegister.xml#IC_101"
-                                           codeListValue="theme" />
+                                          codeListValue="theme"/>
```

Comment:
This is incorrect.
We have the following PR to fix the issue but still waiting on Approval from Geocat?
https://github.com/metadata101/iso19139.ca.HNAP/pull/345

If you make this change, it will only cause conflicts
Is it possible to approve and merge that PR first and then update this file?

Reply:
Yes of course.